code/unclassified/lapindrom_checker/lapindrome_checker.cpp

```diff
@@ -0,0 +1,35 @@
+// part of cosmos by opengenus foundation
+
+#include <iostream>
+#include <algorithm>   // used for sort()
+using namespace std;
+
+bool is_lapindrome(string str)
+{
+  int mid = str.length() / 2;
+  int delim = (str.length() % 2) == 0 ? mid : mid + 1;
+
+  string first_half = str.substr(0, mid);
+  string second_half = str.substr(delim);
+
+  sort(first_half.begin(), first_half.end());
+  sort(second_half.begin(), second_half.end());
+
+  return first_half == second_half;
+}
+
+int main()
```

Comment:
please try to separate test file and source file, and try to use Catch2 which is powerful tool for c++ test. (check root/test of directory)